text/0000-adoption-of-a-rustacean-crustacen.md

```diff
@@ -0,0 +1,86 @@
+- Feature Name: Adopt Ferris as the Official Mascot/Epoch Costumes
+- Start Date: 2018-02-01
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Officially adopt Ferris as the mascot for the current epoch and have a new
```

Comment:
IMO we can keep Ferris as the mascot for _Rust_ itself, and have an epoch mascot as well.

Reply:
I agree with @Manishearth; We can use prior art by the android open source project (ASOP) and give each epoch a code-name as well as a new look for Ferris based on the code-name...

Reply:
@Manishearth this was another thing I had considered, but had opted for ferris dressed up with each epoch. Mostly because I was unsure if people would be receptive to more than one mascot. Prior not released wordings had the reasoning that it's what an epoch is "Newer versions playing well with older ones" and each one Ferris gets a new friend to play with.

Reply:
Idk, Ferris is already quite established as the Rust community mascot, retiring them because it's only the 2015 epoch doesn't seem great to me.